src/components/primitive/Icons/icons.ts

```diff
@@ -0,0 +1,10 @@
+import { arrowRight, arrowLeft, arrowDown, hamburgerMenu } from '../../../assets';
```

Comment:
No direct imports. Use theme instead. Add an 'icons' property whose value can be accessed based on the key passed as a prop.